regression/esbmc/time_02/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.c
+
+^VERIFICATION FAILED$
```

Comment:
Why is this program failing?

Reply:
As time() function expects time_t pointer, if an int is passed it results in a out of bound.

Counterexample:

State 1 file time.c line 16 function time thread 0
----------------------------------------------------
Violated property:
  file time.c line 16 function time
  dereference failure: Access to object out of bounds

VERIFICATION FAILED